### ISSUE TITLE
Voice Chat Testing

### ISSUE THREAD

**Issue by opikropikin** [XrunMeta/CLUBX#98]
there is a bug found, when the user has been in the area of the night club. then there are new users who have just joined, the user button that is currently active in direct voice chat becomes gray as if it is not active, it should remain Yellow which means it is active. please check

**Comment by trisutrisnowapu** [comment]
noted I will check this issue

**Comment by opikropikin** [comment]
okey three thankyou

**Comment by trisutrisnowapu** [comment]
I can reproduce this issue, this one is complex since it connected to server logic and only affect to players connected to private chat. It will take some time to investigate this issue and fix. But I will start checking it

**Comment by trisutrisnowapu** [comment]
I'm starting to working on this

**Comment by trisutrisnowapu** [comment]
This is pretty complex as the users list is cleared up and newly regenerated (with addition of 1 new user who is joining) each time a new user joins. The mic icon is local logic, while users list is a server logic and multicasted to all users in the server. I need to find out a way to search which one is joining (widget is in the local instance of the user, since the server instance of the user does not own this information), and I have to test this on minimum of 3 phone to do this. 2 phone needs to be conencted with private voice, and after that 1 phone is enter the server and see whether the fix I apply is working or not

**Comment by opikropikin** [comment]
yes it looks simple in function, but in this program it is very complicated and takes time, let's solve it and test it repeatedly

**Comment by opikropikin** [comment]
morning, tri, how it ... about icon active in inactive
